docs/howto/windows.md

```diff
@@ -106,28 +87,132 @@ many of these.)
   but it may be a bit simpler to use the Linux version.
 * One significant limitation of WSL (as of 2018) is that doing a large
   amount of filesystem I/O tends to be slow.  This is a reason to do
-  operations like a Gradle build, perhaps `git` commands, and perhaps
+  operations like a Gradle build ( Gradle is the Java program that does the Android build), perhaps `git` commands, and perhaps
   the occasional `yarn install` using the Windows versions.
 
 [so-guide]: https://stackoverflow.com/questions/42614347/running-react-native-in-wsl-with-the-emulator-running-directly-in-windows
 
-### Accessing the filesystem
-You can access your Windows local drives from `bash` and other Linux
-programs by navigating to `/mnt/<drive-letter>` .
 
-### Running `yarn` commands
-First, install Node.js and Yarn in your Linux environment.  You'll follow
-the same steps as in [our main setup guide](build-run.md) -- just use the
-Linux versions. :-)
+### Step-by-step instructions for setting up WSL
```

Comment:
Awesome, thanks for these detailed step-by-step instructions!

Can you say more about how you worked out these steps? Were there existing instructions online that you were able to use for parts of them? Is it sort of a cleaned-up transcript of the steps you ended up taking to set things up?

The answer to how you worked them out doesn't necessarily go in the doc, but I definitely want to know :) and it probably goes in the commit message too. One reason is that inevitably there will be things that don't work for some people, and then it'll be super helpful to know why that step was there as we're figuring out how to fix it for them. Also, the right instructions will change over time, so knowing why the different pieces are there will again be super helpful for keeping it up to date with changes in WSL and the various tools we install.

Reply:
Not a problem. I figured the steps through a combination of the initial draft that was already here, and then searching for answers whenever I encountered an error on the way. A lot of the steps in the documentation are therefore just hedges. Step 4 is a case in point. One one computer I worked on while testing these instructions, some of the steps in 4 were not needed. However,  a different Windows machine required every single step before it was happy to proceed.